modules/overlays/test/overlays/html-tooltip-overlay.test.js

```diff
@@ -0,0 +1,17 @@
+// @flow-ignore
+import React from 'react';
+import Renderer from 'react-test-renderer';
+
+import { viewport } from '../../../../core/src/test/mocks';
+import Nebula from '../../lib/nebula-react';
```

Comment:
Tip: We import `aliases.js` in test files (and set them with `module-alias`) so that we can import using the right paths.